internal/cmd/egctl/testdata/translate/out/invalid-envoyproxy.all.yaml

```diff
@@ -38,6 +38,34 @@ gatewayClass:
       reason: InvalidParameters
       status: "False"
       type: Accepted
+    supportedFeatures:
+    - GRPCRoute
+    - Gateway
+    - GatewayHTTPListenerIsolation
```

Comment:
the skip test values still showed up here, they shouldnt

Reply:
Just to make sure I understand:
We have a list of `SkipTests.` Should we delete the `Features` of every skipped test from the final supported features list?

Reply:
I also noticed that the list of `MeshExtendedFeatures` appears in the supported features. This happens since we only specify `MeshCoreFeatures` in the `ExemptFeatures` list. It doesn't make sense (correct me if I'm wrong) to have the extended without the core.

Should I add the `MeshExtendedFeatures` to the `ExemptFeatures` list?

Reply:
Yes we should be deleting them from this list because we don't support it yet :) 

Sure let's also MeshExtendedFeatures to ExemptFeatures

Reply:
Thanks for the clarification @arkodg.  I've added the requested change.
Please notice though that features such as `Gateway` and `HTTPRoute` have been removed (as a dependency of skipped test named `GatewayHTTPListenerIsolation`).

Reply:
thanks for highlighting that, that is something worth discussing in upstream since `GatewayHTTPListenerIsolation` is an extended test